index.html

```diff
@@ -120,67 +135,130 @@ <h3 class="tittle3">Finish mobile section 5</h3>
+                </div>
 
-        <section id="languages2">
-            <h4>Frameworks</h4>
+                <div class="langrid">
+                     <div id="languages">
+                        <h4>Languages</h4>
 
-            <ul class="ul1">
-                <li>Bootstrap</li>
-                <li>Ruby on rails</li>
-                <li>RSpec</li>
-                <li>Capibara</li>
-                <li>Selenium</li>
+                        <ul class="ul1">
+                            <li>JavaScript(ES5/ES6)</li>
+                            <li>Ruby</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
 
-            </ul>
+                        </ul>
 
-        </section>
+                    </div>
 
-        <section id="languages3">
-            <h4>Skills</h4>
+                    <div id="languages2">
+                        <h4>Frameworks</h4>
+
+                        <ul class="ul1">
+                            <li>Bootstrap</li>
+                            <li>Ruby on rails</li>
+                            <li>RSpec</li>
+                            <li>Capibara</li>
+                            <li>Selenium</li>
+                        </ul>
+
+                    </div>
+
+                    <div id="languages3">
+                        <h4>Skills</h4>
+
+                        <ul class="ul1">
+                            <li>Codekit</li>
+                            <li>Codepen</li>
+                            <li>GitHub</li>
+                            <li>GitLAb</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+
+                    <div id="languages4"></div>
+                </div>
 
-            <ul class="ul1">
-                <li>Codekit</li>
-                <li>Codepen</li>
-                <li>GitHub</li>
-                <li>GitLAb</li>
-                <li>Terminal</li>
-            </ul>
         </section>
 
-        <section id="greetings2">
-            <h1 class="GE">Get started</h1>
-            <p class="firstp">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque doloribus maiores,
-                rem excepturi.</p>
-                
+        <section id="initial2"> 
+        
+            <img class="left" src="./images/getstartedl.svg" height="263px" alt="left">
+                        
                  <form class="form" action="https://formspree.io/f/xjvzpoww" method="post">
-                    <ul>
-                        <li><label for="text">Your name</label>
+
+                    <h1 class="GE">Get started</h1>
+
+                    <p class="firstpGE">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+                    
+                    <ul id="savedata">
```

Comment:
- [ ] please ensure that the inputs are well aligned with the contact text so it looks like that of figma
![Screenshot (576)_LI](https://user-images.githubusercontent.com/65298048/199896725-eafe9d3a-86d5-411d-86bf-288450bfeda9.jpg)